pylint/checkers/base/basic_error_checker.py

```diff
@@ -46,7 +46,7 @@ def _get_break_loop_node(break_node: nodes.Break) -> nodes.For | nodes.While | N
     return parent
 
 
-def _loop_exits_early(loop: nodes.For | nodes.While) -> bool:
+def loop_exits_early(loop: nodes.For | nodes.While) -> bool:
```

Comment:
Let's keep our private functions private, we don't want to have to deprecate things if people assume we voluntarily made this public. The old functions that are not explicitly private are not public either but at least the current maintaining team did not make them private herself.